crates/cli-support/src/intrinsic.rs

```diff
@@ -102,9 +105,75 @@ intrinsics! {
         #[symbol = "__wbindgen_is_string"]
         #[signature = fn(ref_externref()) -> Boolean]
         IsString,
+        #[symbol = "__wbindgen_is_bigint"]
+        #[signature = fn(ref_externref()) -> Boolean]
+        IsBigint,
+        #[symbol = "__wbindgen_typeof"]
+        #[signature = fn(ref_externref()) -> Externref]
+        Typeof,
+        #[symbol = "__wbindgen_in"]
+        #[signature = fn(ref_externref(), ref_externref()) -> Boolean]
+        In,
         #[symbol = "__wbindgen_is_falsy"]
         #[signature = fn(ref_externref()) -> Boolean]
         IsFalsy,
+        #[symbol = "__wbindgen_as_number"]
+        #[signature = fn(ref_externref()) -> F64]
+        AsNumber,
+        #[symbol = "__wbindgen_try_into_number"]
+        #[signature = fn(ref_externref()) -> Externref]
+        TryIntoNumber,
+        #[symbol = "__wbindgen_neg"]
+        #[signature = fn(ref_externref()) -> Externref]
+        Neg,
+        #[symbol = "__wbindgen_bit_and"]
+        #[signature = fn(ref_externref(), ref_externref()) -> Externref]
```

Comment:
These bitwise operators always return an i32, besides unsigned_shr, which yields u32.
They're currently typed as yielding Externref.

But, on another note, these are already native Rust operators, why use the significantly slower, morr unsound JS operators? Is there anything that can't be done without these?

Reply:
These operators only return 32-bit ints when passed JS `number`s. When passed `bigint`s they return `bigint`s. (The only exception is `>>>` (`unsigned_shl`), which does not accept `bigint`s; I included it for completeness but if it's deemed unnecessary I can remove it